### DISCUSSION TITLE
针对youtube-dl的改变

### DISCUSSION THREAD

**Discussion by donald-ning** [iina/iina#3782] [General]
对于youtube-dl这个插件来说，他的下载速度已无法满足日常需求，能否将youtube-dl改为yt-dlp，这样可能会更好一些

**Comment by low-batt** [comment]
以英语为母语的人使用在线翻译器进行回复。
希望这个翻译在某种程度上是可以理解的...

刚刚发布的 IINA 1.3.0 嵌入了 yt-dlp 而不是 youtube-dl。IINA应用程序捆绑包中的 "youtube-dl "实际上是yt-dlp。

```bash
low-batt@gag MacOS$ /Applications/IINA.app/Contents/MacOS/youtube-dl --version
2022.05.18
low-batt@gag MacOS$ /Applications/IINA.app/Contents/MacOS/youtube-dl --help
Usage: yt-dlp [OPTIONS] URL [URL...]

Options:
  General Options:
    -h, --help                                          Print this help text and exit
...
```

IINA 1.3.0可以从[IINA](https://iina.io/)网站上下载。

IINA以前的版本使用的是一个旧版本的youtube-dl。那个旧版本的youtube-dl在访问YouTube方面肯定有问题。
